documentation/modules/ROOT/pages/setup.adoc

```diff
@@ -1,6 +1,6 @@
 [[requirements]]
 = Setup
-include::../_attributes.adoc[]
```

Comment:
Not sure about this, but with the brackets I couldn't build the project locally.